clients/imodels-client-management/tsconfig.esm.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Looks like it would be good to have base esm config in utils package. This way I assume we would not need to have `moduleResolution` flag in cjs config and we could share `module`, `target` properties across specific esm configs in packages.